### DISCUSSION TITLE
question on “IEA15MW_01”.

### DISCUSSION THREAD

**Discussion by liangzhi85** [IEAWindTask52/LidarAssistedControl#18] [Q&A] [answered]
Dear IEA LAC team,
Thank you very much for developing and sharing this interesting tool!
I'm still learning this tool and try to modify a little bit.

My questions is blew:
1. I try to extend the measurement distance further (RangeGates-Tab in “SinglePoint_LidarFile.dat”) to see what happens. 
2. I feel the turbine blade follows the wind signal but pitches earlier, so I test the line “3.0       ! T_buffer” in the file “FFP_v1.IN”.
3. I don't know if I understand right. Please see my changes and the figures below.
4. Could you please make some comment? Thanks in advance!

My modification and the figures:
1)	The regular result by run the “IEA15MW_01”
        ![image](https://github.com/IEAWindTask52/LidarAssistedControl/assets/123851926/830ff217-46d0-4bcd-a998-60be307f30e5)
2)	I try to change “RangeGates-Tab = 60 ” to “RangeGates-Tab = 120”, in the file “SinglePoint_LidarFile.dat”
        ![image](https://github.com/IEAWindTask52/LidarAssistedControl/assets/123851926/3fa99519-dc52-4ab2-8a3b-081f8df8ef54)
3)	I continue try to change “3.0       ! T_buffer” to “8.0           ! T_buffer” in the file “FFP_v1.IN”
        ![image](https://github.com/IEAWindTask52/LidarAssistedControl/assets/123851926/4a5095f2-cde3-46f2-9710-ab05d6fcc7b7)
4)	I continue try to change “8.0       ! T_buffer” to “13.0         ! T_buffer” in the file “FFP_v1.IN”
        ![image](https://github.com/IEAWindTask52/LidarAssistedControl/assets/123851926/c893fafe-7fad-4b59-a037-e5e37ceb55fb)

**Comment by SimonWeich** [comment, marked as answer]
Dear Zhi Liang,

Thanks again for participating in this year's LAC Summer Games! Here is a summary of what we talked about after the last LAC-meeting:

In general, it is possible to increase measuring distance (SinglePoin_LidarFile.dat -> "RangeGates-Tab" [m]). But it would be necessary for you also to change the buffer time (FFP_v1.IN -> "T_buffer" [s]) accordingly. That is the case because you must take the time the wind needs to travel from the measuring point to the turbine (Taylor's Frozen Turbulence Hypothesis is applied here). Filter delays may also influence your required buffer time.

For further reference, you can take a look into David's PhD thesis: "Lidar-Assisted Control Concepts for Wind Turbines" (e.g. chapter 5.5.2).

Don't hesitate to approach us should you have any further questions.

Best regards
Simon

**Reply by liangzhi85** [reply]
Great, thank you very much!

**Comment by Javier-TRI** [comment]
Dear all,

I have a question about the example provided. Is there a corresponding ROSCO .yaml file available that can be used to reproduce the provided .IN file?

Alternatively, could you please explain how the .IN file was generated?

Thank you,

Javi

**Reply by DavidSchlipf** [reply]
Hi Javie,
The yaml file you might find in the IEA-15-240-RWT repository. The origin of the IN file is documented in the "ChangeLog.txt" in the example folders:

The ROSCO input file is based on
https://github.com/IEAWindTask37/IEA-15-240-RWT/blob/master/OpenFAST/IEA-15-240-RWT-Monopile/ServoData/DISCON-Monopile.IN
using the latest version
dzalkind: Update Cp file paths, Cp table, tested locally, d0821e6, on Apr 5
https://github.com/IEAWindTask37/IEA-15-240-RWT/tree/d0821e67e4ae808a76d24690e05575cd2be11942
and has been modified to run with ROSCO v2.6 by using additional lines from
https://github.com/NREL/ROSCO/blob/main/Test_Cases/IEA-15-240-RWT-UMaineSemi/DISCON-UMaineSemi.IN
including Cp_Ct_Cq.IEA15MW.txt

Hope this helps!
David
